### ISSUE TITLE
"I'm not a robot" Google Recaptcha 2 descriptive text is not read by NVDA in IE 11

### ISSUE THREAD

**Issue by nileshpenta** [nvaccess/nvda#7038]
NVDA is not reading out "I'm not a robot" descriptive text in Google Recaptcha 2 widget when we tab to the checkbox. Reading fine in Firefox and Chrome. Please check.

**Edit:** The following screenshot shows the google reCAPTCHA on a web page, which has an unchecked checkbox next a label of "I'm not a robot". It has the reCAPTCHA logo, with two links under the log, "privacy" and "terms" . Speech viewer is also visible which has the following text highlighted:
```
check box not checked
Privacy
link
Terms
link
out of frame
CONTINUE button
```
Before the highlighted section is the URL of the reCAPTCHA, and followed by the text "Recaptcha requires verification"
The following shows the google recaptcha in use: https://patrickhlauke.github.io/recaptcha/

![image](https://cloud.githubusercontent.com/assets/656815/24560197/b07b8838-160f-11e7-9242-0ee5b36dc1db.png)

**Comment by PratikP1** [comment]
This I can verify based on the examples provided in #7037

**Comment by feerrenrut** [comment]
Updated the description of this issue to properly explain what is in the screenshot. @nileshpenta please note that many of the contributors to NVDA are blind and would not gain much information from this screenshot.

When testing this myself, ( on recent `next` build of nvda with IE11) I can verify the following
**In focus mode** and using tab to move to the check box, "I'm not a robot  check box  not checked" is announced. 
**In browse mode** and using the down arrow to move to the checkbox and beyond, I can verify that "I'm not a robot" is not announced.

This behaviour likely comes from the `aria-hidden` in the following snippet:
```
<label class="rc-anchor-center-item rc-anchor-checkbox-label" id="recaptcha-anchor-label" role="presentation" aria-hidden="true">
  <span aria-labelledby="recaptcha-accessible-status" aria-live="polite"></span>I'm not a robot
</label>
```

Setting `aria-hidden` to `false` in the above snippet changes the behaviour to match the following:
While in browse mode, after arrow-down after the check box the following is announced. 
"Recaptcha requires verification. I'm not a robot"

As mentioned in the initial description, this doesn't happen in other browsers. We will need to inspect our handling of this for IE and see how it is different from chrome / firefox.

**Comment by Adriani90** [comment]
This is still reproducible in NVDA 2019.1.
